### PR TITLE
Feature/geocoding language local admin updates

### PR DESCRIPTION
**Pull request by benetis**
For discussions. Before merging needs testing + API updates to land on prod

### CODE REVIEW

File: shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala [hunk 1]
```diff
@@ -78,4 +115,6 @@ object RequestUtils {
       case a: JsError => ValidationError(a)
     }
   }
+
```
Comment (Bloznelis) [inline shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala]: Lotta space

File: shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/geocoding/ReverseGeocodingRequest.scala [hunk 2]
```diff
@@ -17,17 +17,19 @@ import com.igeolise.traveltimesdk.json.reads.GeocodingReads._
   *                      Useful when performing reverse geocoding requests near a border.
   */
 case class ReverseGeocodingRequest(
-  coordinates: Coords,
-  withinCountry: Option[String]
-) extends TravelTimePlatformRequest[GeoJsonResponse[GeocodingResponseProperties]] {
+                                    coordinates: Coords,
```
Comment (Bloznelis) [inline shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/geocoding/ReverseGeocodingRequest.scala]: Maybe we should maintain consistent indedtation.

File: shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala [hunk 3]
```diff
@@ -25,6 +26,42 @@ object RequestUtils {
     request: Request[String, S]
   )
 
+  def sendGeocoding[R[_]: Monad, S](
```
Comment (Bloznelis) [inline shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala]: Feels too concrete here. Suggestion:
```scala
def sendModified[R[_]: Monad, S, Response](
    sttpRequest: SttpRequest[R, S],
    modifyResponseFn: STTP.Response[String] => Either[TravelTimeSdkError, Response]
  ): R[Either[TravelTimeSdkError, Response]] = {

    sttpRequest
      .backend
      .send(sttpRequest.request)
      .map(modifyResponseFn)
  }

  def addLanguageToResponse(
    validationFn: JsValue => JsResult[GeoJsonResponse[GeocodingResponseProperties]]
  ): STTP.Response[String] => Either[TravelTimeSdkError, GeocodingResponse] = response => {

    def findContentLanguageHeader(headers: Seq[(String, String)]): BCP47 = {
      headers.find(_._1 == "Content-Language") match {
        case Some(value) => BCP47(value._2)
        case None => BCP47("en")
      }
    }

    response
      .body
      .handleJsonResponse(validationFn)
      .map { properties =>
        GeocodingResponse(
          language = GeocodingLanguageResponse(findContentLanguageHeader(response.headers)),
          properties = properties
        )
      }
  }
```

Comment (benetis) [reply shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala]: sounds good

File: shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala (hunk 1 above)
Comment (benetis) [reply shared/src/main/scala/com/igeolise/traveltimesdk/dto/requests/RequestUtils.scala]: i did scalafmt reformat before commit. Maybe we need to adjust the settings?

**Pull request merged**